mobile/lib/entities/asset.entity.dart

```diff
@@ -297,13 +304,15 @@ class Asset {
           width: a.width ?? width,
           height: a.height ?? height,
           exifInfo: a.exifInfo?.copyWith(id: id) ?? exifInfo,
+          altText: a.altText ?? altText,
         );
       } else if (isRemote) {
         return _copyWith(
           localId: localId ?? a.localId,
           width: width ?? a.width,
           height: height ?? a.height,
           exifInfo: exifInfo ?? a.exifInfo?.copyWith(id: id),
+          altText: altText ?? a.altText,
```

Comment:
I'm not sure how to reproduce the scenario in this conditional, but the alt text coming from the remote copy should always be preferred.